src/core/components/AMInstallButton/index.js

```diff
@@ -130,7 +140,24 @@ export class AMInstallButtonBase extends React.Component<InternalProps> {
   };
 
   installExtension = async (event: SyntheticEvent<HTMLAnchorElement>) => {
-    const { addon, enable, install, isAddonEnabled } = this.props;
+    const {
+      _tracking,
+      addon,
+      enable,
+      install,
+      isAddonEnabled,
+      variant,
+    } = this.props;
+
+    const category = `${EXPERIMENT_CATEGORY_CLICK}-${
```

Comment:
We should probably restrict this newly added code to `addon.type === ADDON_TYPE_EXTENSION` only, no?